subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -345,7 +345,7 @@ ull_conn_iso_lll_stream_sorted_get_by_group(struct lll_conn_iso_group *cig_lll,
 		cis = ll_conn_iso_stream_get(handle);
 
 		/* Match CIS contexts associated with the CIG */
-		if (cis->group == cig) {
+		if ((cis->group == cig) && cis->lll.active) {
```

Comment:
Since `cis->lll.active` is technically not a boolean, although treated as such, it does not really match our coding guidelines that require control statements like `if`s to use effective boolean expressions. 

It's just a mention, as I don't know if this is something that should/could be applied in the controller code, but our coding guide does _technically_ require this to be
```suggestion
		if ((cis->group == cig) && cis->lll.active == 1U) {
```